src/hpnmg/ProbabilityCalculator.h

```diff
@@ -70,6 +75,8 @@ namespace hpnmg {
 
 		static double transformedFunctionToIntegrateMonteCarlo(double *k, size_t dim, void* params);
 
+        static double functionToDirectIntegrateMonteCarlo(double *k, size_t dim, void* params);
```

Comment:
We should start adding more Documentation:
- GSL integrand for `getProbabilityForUnionOfPolytopesUsingDirectMonteCarlo`
- `params` is of type `functionToDirectIntegrateMonteCarloParams`
- evaluates to the product of the pdfs provided in `params`, or `0` if `k` is not within the union of polytopes in `params`.